scripts/run-python-tests.sh

```diff
@@ -10,4 +10,4 @@ set -eu
 scriptdir=$(readlink -f "$(dirname "$0")")
 cd "$scriptdir/.."
 
-pytest -svx source/Mlos.Python
+pytest -svxl source/Mlos.Python $*
```

Comment:
The `-l` may not be necessary.  We had `--locals` in the previous incantation thought ...